DevCycle.SDK.Server.Common/Model/Local/EventTypes.cs

```diff
@@ -2,7 +2,9 @@ namespace DevCycle.SDK.Server.Common.Model.Local
 {
     public static class EventTypes
     {
+        public static readonly string aggVariableDefaulted = "aggVariableDefaulted";
         public static readonly string variableDefaulted = "variableDefaulted";
```

Comment:
We can probably remove the individual variable evaluated now - since we only aggregate.